frontend/src/metabase/search/utils/user-search-params/user-search-params.ts

```diff
@@ -0,0 +1,21 @@
+import type { UserId } from "metabase-types/api";
+import type { SearchQueryParamValue } from "metabase/search/types";
+
+export const parseUserIdString = (
```

Comment:
Moved from `CreatedByFilter.fromUrl` and `CreatedByFilter.toUrl` component to `parseUserIdString` and `convertUserIdToString`